app/controllers/groups_controller.rb

```diff
@@ -47,7 +46,15 @@ def remove_group
         @removed_groupings.push(grouping)
       end
     end
-    head :ok
+    unless @errors.empty?
+      err_msg = 'The following groupings could not be deleted since they have submissions: ' +
```

Comment:
This is a very good error message. You should move this to the localization files, by adding a new key (maybe under `views/groups/`). You can add the corresponding keys for each language, but use "UPDATE ME" for the other languages.